### ISSUE TITLE
Texter blocks drag n drop (?)

### ISSUE THREAD

**Issue by limbo666** [adampash/texter#4]
I'm pretty sure that texter causes a strange behaviour on my systems.
Under some (unknown or better unclear) circumstances the system refuses the drag n drop function.
I have experienced this issue in three computers with the same bundles and the latest version of texter available. Operator systems Win 7 Ultimate, Win 7 Starter and Win XP.
Can somebody confirm this issue?
 I goggled it and its a know issue to others as well: 
http://www.google.co.uk/#hl=en&source=hp&q=texter+drag+n+drop&aq=f&aqi=g-sx1&aql=&oq=&gs_rfai=&fp=4decbba4b4ffde8d

**Comment by leeand00** [comment]
Yes I can confirm this, I've experienced this on Vista-64 and Windows 7-64.   The only way around it seems to be if you click on the desktop, and press escape several times and then go back and try to continue the drag and drop operation.  

I'm not really sure why this is, maybe adam would know.

**Comment by jblaine** [comment]
This still exists in 0.6.0 as of today. Makes usage the tool largely a non-starter. It would seem this is a largely abandoned project.

**Comment by wolfkin** [comment]
I think I had this issue but solved it by turning off "Universal Spelling Auto Correct"

Just a note for anyone in the future who might have this issue
